tests/queries/0_stateless/02841_group_array_sorted.sql

```diff
@@ -0,0 +1,22 @@
+SELECT groupArraySorted(5)(number) from numbers(100);
+
+SELECT groupArraySorted(number) from numbers(100);
+
+SELECT groupArraySorted(100)(number) from numbers(1000);
+
+SELECT groupArraySorted(5)(str) FROM (SELECT toString(number) as str FROM numbers(10));
```

Comment:
For strings, I would find it more interesting to test `groupArraySorted()` on numbers 0-30 as string. If I see correct (didn't test it), it will sort lexicographically (`1, 10, 11, ..., 19, 2, 20, 21, ...`) instead of numerically.